spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/source/SparkPositionDeltaWrite.java

```diff
@@ -483,6 +493,14 @@ protected PartitioningWriter<PositionDelete<InternalRow>, DeleteWriteResult> new
         SparkFileWriterFactory writers,
         OutputFileFactory files,
         Context context) {
+      TableOperations ops = ((HasTableOperations) table).operations();
+      Function<CharSequence, PositionDeleteIndex> previousDeleteLoader =
```

Comment:
What about a helper method like `createPreviousDeleteLoader` or `PreviousDeleteLoader.create`?

Reply:
I like the idea of a static util, added!